tests/kallax.go

```diff
@@ -3120,29 +3162,32 @@ func (rs *PersonResultSet) Close() error {
 
 // NewPet returns a new instance of Pet.
 func NewPet(name string, kind string, owner *Person) (record *Pet) {
-	record = newPet(name, kind, owner)
-	if record != nil {
-		record.SetID(kallax.NewID())
-	}
-	return
+	return newPet(name, kind, owner)
```

Comment:
I think that every brand new model that `!schema.autoIncr` should have its own ID because Model fulfill the `IDSetter` interface, shouldn't they?